and-cli-deploy-azure-storage.test.js

```diff
@@ -0,0 +1,137 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const { ERROR_OUTPUT_STRING }   = require("./_modules/constants")
+const { shouldDisplayHelpMenu } = require("./tests/describes");
+const testUtils                 = require("./tests/test-utils");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("and-cli-deploy-azure-storage", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region username
+    // -----------------------------------------------------------------------------------------
+
+    describe("username", () => {
```

Comment:
Suggestion: Just have one other `username` describe block around all 3 tests instead of repeating it